modules/afsocket/afinet-dest.c

```diff
@@ -547,6 +547,18 @@ afinet_dd_spoof_write_message(AFInetDestDriver *self, LogMessage *msg, const Log
 }
 #endif
 
+static inline gboolean
+_is_message_spoofable(LogMessage *msg)
+{
+  return msg->saddr && (msg->saddr->sa.sa_family == AF_INET || msg->saddr->sa.sa_family == AF_INET6);
+}
+
+static inline gboolean
+afinet_dd_is_spoof_source_enabled(AFInetDestDriver *self)
```

Comment:
Just a minor inconsistency, `_is_message_sppofable` starts with underscore while this is not. Probably better start both with underscore. Especially, the naming of `afinet_dd_is_spoof_source_enabled` resembles to a public api name.  Probably `_is_spoof_source_enabled` would be enough.